rc/lsp.kak

```diff
@@ -1734,6 +1740,56 @@ define-command -hidden lsp-show-message-log -params 1 -docstring %{
     echo -debug "kak-lsp: log:" %arg{1}
 }
 
+define-command -hidden lsp-show-message-request -params 4.. -docstring %{
+    lsp-show-message-request <prompt> <on-abort> <opt> <command> [<opt> <command>]...
+    Render a prompt message with options.
+} %{
+    evaluate-commands -try-client %opt{toolsclient} %{
+        info -title "From LSP" %arg{1}
+        evaluate-commands %sh{
+            on_abort="$2"
+            shift 2
+            echo lsp-menu -on-abort $on_abort "$@"
```

Comment:
I'm passing the args naked here, so I had to quote twice on kak-lsp side, I wonder if there's any better way to do it?

Reply:
the double Kakoune-quoting makes sense, because we evaluate the `lsp-menu` command here and later the `lsp-menu` command will evaluate its args once again.
It's not super pretty but it saves processes so it's okay.
We do still want to quote `$on_abort` to protect against shell evaluation, which is unrelated to Kakoune quoting/evaluation